### PR TITLE
Configure predictions after AWSMobileClient

### PR DESCRIPTION
**Pull request by wooj2**
Generally we advise to configure amplify after AWSMobileClient is fully initialized.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/predictions/fragments/ios/getting-started.md [hunk 1]
```diff
@@ -137,13 +137,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 		}
 
 		print("AWSMobileClient initialized, userstate: \(userState)")
```
Comment (kneekey23) [inline docs/lib/predictions/fragments/ios/getting-started.md:139]: Shouldn’t we add the `AWSMobileClient.initialize` call with the print inside the closure?

Comment (wooj2) [reply docs/lib/predictions/fragments/ios/getting-started.md:139]: Sorry... hmm not following what you are suggesting?

Comment (kneekey23) [reply docs/lib/predictions/fragments/ios/getting-started.md:139]: ignore, i am dumb

Comment (wooj2) [reply docs/lib/predictions/fragments/ios/getting-started.md:139]: No you're not! :)

**Comment by brene** [comment]
@jpignata / @mlabieniec - I mistakenly merged this without your 👍 , can you review and give a thumbs up?

**Pull request merged**